pkgs/applications/misc/go-jira/default.nix

```diff
@@ -0,0 +1,23 @@
+# This file was generated by https://github.com/kamilchm/go2nix v1.2.1
+{ stdenv, buildGoPackage, fetchgit }:
+
+buildGoPackage rec {
+  name = "go-jira-${version}";
+  version = "1.0.17";
+  rev = "b98da3612d6ab0f3872b64ff36e6cf93219fb4d4";
```

Comment:
`rev` is then useless

Reply:
Thanks @nlewo :) This has been amended now.